ares/n64/cic/cic.cpp

```diff
@@ -8,9 +8,16 @@ CIC cic;
 #include "serialization.cpp"
 
 auto CIC::power(bool reset) -> void {
-  model = cartridge.node ? cartridge.cic() : dd.cic();
+  if(Model::Aleck64()) {
+    model = "CIC-NUS-5101";
+  } else {
+    model = cartridge.node ? cartridge.cic() : dd.cic();
+  }
```

Comment:
Is this necessary? Is it possible to just put the CIC string in the pak like we do for n64?

Reply:
I suppose we could, but arcade titles don’t go through the n64 pak